packages/clerk-js/src/ui.retheme/elements/PhoneInput/PhoneInput.tsx

```diff
@@ -169,7 +178,13 @@ const CountryCodeListItem = memo((props: CountryCodeListItemProps) => {
       ]}
       {...rest}
     >
-      <Flag iso={country.iso} />
```

Comment:
Now that we removed this, you need delete the Flag component as the package is not building